internal/host/validator.go

```diff
@@ -1003,43 +1013,120 @@ func (v *validator) hasSufficientPacketLossRequirementForRole(c *validationConte
 	case ValidationSuccess:
 		return status, "Packet loss requirement has been satisfied."
 	case ValidationFailure:
-
-		return status, fmt.Sprintf("Packet loss percentage requirement of %s %.2f%% not met for connectivity between %s and%s.", comparisonBuilder(*c.clusterHostRequirements.Total.PacketLossPercentage), *c.clusterHostRequirements.Total.PacketLossPercentage, c.host.ID, strings.Join(hostPacketLoss, ","))
+		// When logging, make sure the full timing metrics are logged.
+		fullHostTimingMetrics := v.summarizeHostTimingMetrics(hostMetrics, false)
+		v.log.Info(fmt.Sprintf(`A total packet loss above the tolerated threshold of %.2f%% was encountered when performing connectivity validation between host %s and %s\n`,
```

Comment:
If we log this under `ValidationFailure`, shouldn't this be a `log.Error`?
@filanov what do you think?

Reply:
I agree and have made this change